src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+
+  const convertedObj = {};
+
+  sourceString.split(';').map((elem) => {
```

Comment:
[CODE KNOWLEDGE] - You are using `map` method but not using its return value. The `map` method returns a new array and does not change the original array. In your case, you are not using the new array returned by `map` method. Instead you should use `forEach` method.